### ISSUE TITLE
Actor Generation (PC)

### ISSUE THREAD

**Issue by patrickmohrmann** [patrickmohrmann/earthdawn4eV2#15]
## Reason

to ease up the charakter generation process it would be nice to have everything added by default

since the discipline information comes from the Advancement sheet, this. needs to be finished first.

the **Project** for Actor Generation (PC): 

[V0.0.5 Actor Generation](https://github.com/users/patrickmohrmann/projects/20/views/1)

## Proposed Solution

- [x] #108
- [x] #93
the pop up will guide through the following process:
- [x] #94
- [x] #95
- [x] #96
- [x] #97
- [x] #98
- [x] #99
- [x] #100
- [x] #101
- [x] #102
- [x] #103
- [x] #104
- [x] #105
- [x] #106
  - [x] #107 
- [x] #109

## Technical Details

there shall be certain checks for evaluation.
- [x] Attribute values in check (min -3 max + 8 and not possible to use up more points than in teh setting)
- [x] check if all 8 talent ranks are used 
- [x] check if all skill ranks are used 
- [x] check if languages are chosen

**Comment by patrickmohrmann** [comment]
done
